mapie/tests/test_regression.py

```diff
@@ -517,19 +563,16 @@ def test_results_prefit_naive() -> None:
 
 def test_results_prefit() -> None:
     """Test prefit results on a standard train/validation/test split."""
-    X_train_val, X_test, y_train_val, y_test = train_test_split(
-        X, y, test_size=1 / 10, random_state=1
-    )
-    X_train, X_val, y_train, y_val = train_test_split(
-        X_train_val, y_train_val, test_size=1 / 9, random_state=1
+    X_train, X_calib, y_train, y_calib = train_test_split(
+        X, y, test_size=1/2, random_state=1
     )
     estimator = LinearRegression().fit(X_train, y_train)
-    mapie_reg = MapieRegressor(estimator=estimator, cv="prefit")
-    mapie_reg.fit(X_val, y_val)
-    _, y_pis = mapie_reg.predict(X_test, alpha=0.05)
+    mapie_reg = MapieRegressor(estimator=estimator, method="base", cv="prefit")
+    mapie_reg.fit(X_calib, y_calib)
+    _, y_pis = mapie_reg.predict(X_calib, alpha=0.05)
     width_mean = (y_pis[:, 1, 0] - y_pis[:, 0, 0]).mean()
     coverage = regression_coverage_score(
-        y_test, y_pis[:, 0, 0], y_pis[:, 1, 0]
+        y_calib, y_pis[:, 0, 0], y_pis[:, 1, 0]
```

Comment:
Why don't we check on the test set?

Reply:
Because we alerady do that in `test_linear_regression_results` for other strategies.
We can also see that we have obtained the same result as the `split` method in the same configuration.
WIDTH: "split": 3.89, "prefit": 3.89,
COVERAGE: "split": 0.956, "prefit": 0.956
Finally, as we are calculating the coverage on the calibration set, we can be sure that the empirical coverage is greater than the target coverage.